### PR TITLE
Add a section about brave browser web socket setting

### LINKED ISSUE #1179: Implement Section concerning `Restrict WebSockets pool` on Brave Browser
Along with #1178 we should create a new section in Landing Page that will explain the Websocket Restrictions that Brave introduced in version 1.3.6 and a step-by-step guide on deactivating the settings.

Something similar to [this](https://docs.talisman.xyz/talisman/help-and-support/troubleshooting/balances-on-brave-not-showing) would do.

### PR DESCRIPTION
**Pull request by wirednkod**
Closes #1179

### CODE REVIEW

**Comment by tomaka** [comment]
What's the difference with https://github.com/paritytech/substrate-connect/pull/1180 ?

**Comment by wirednkod** [comment]
> What's the difference with #1180 ?

This PR adds an informative section about Brave websockets in the Landing page. #1180 has the impl of the modal in the extension's popup and options page.

File: projects/landing-page/src/App.tsx [hunk 1]
```diff
@@ -369,6 +369,41 @@ const App: React.FunctionComponent = () => {
               </SectionRef>
             </CardProject>
           </Section>
+          <Section>
+            <SectionHeading id="troubleshooting" prefix="9">
+              Brave browser web-sockets issue
```
Comment (tomaka) [inline projects/landing-page/src/App.tsx:374]:
```suggestion
              Brave browser WebSocket issue
```

File: projects/landing-page/src/App.tsx [hunk 2]
```diff
@@ -369,6 +369,41 @@ const App: React.FunctionComponent = () => {
...
+            </SectionHeading>
+            <SectionText>
+              As of <b>Brave v1.36</b>, extensions and web pages are limited to
+              a maximum of 10 active web socket connections(More details
```
Comment (tomaka) [inline projects/landing-page/src/App.tsx:378]:
```suggestion
              a maximum of 10 active WebSocket connections (more details
```

File: projects/landing-page/src/App.tsx [hunk 3]
```diff
@@ -369,6 +369,41 @@ const App: React.FunctionComponent = () => {
...
+                target="_blank"
+                rel="noreferrer"
+              >
+                {" "}
+                respective repository{" "}
+              </a>
+              of Brave browser).
+            </SectionText>
+            <SectionText>
+              If an issue is identified concerning web sockets, this setting can
```
Comment (tomaka) [inline projects/landing-page/src/App.tsx:391]:
```suggestion
              If an issue is identified concerning WebSockets, this setting can
```

**Review by tomaka — APPROVED** [review]

**Pull request merged**